_includes/readalong.html

```diff
@@ -0,0 +1,7 @@
+<div class='readalong'>
+    <!-- ReadAlong Web Component -->
+    <link href="https://fonts.googleapis.com/css?family=Lato|Material+Icons|Material+Icons+Outlined" rel="stylesheet">
+    <script src='https://unpkg.com/@roedoejet/readalong@latest/dist/read-along.js'></script>
+    <read-along text='{{include.text}}' alignment='{{include.alignment}}' audio='{{include.audio}}'></read-along>
+    <noscript>Please enable JavaScript to view the ReadAlong</noscript>
+</div>
```

Comment:
This is awesome!

Reply:
yea! I'm kind of annoyed I need to load all the scripts each time the shortcode is included. I'm going to continue to try and find a way to just globally import it on the site, to prevent it from blocking the page loads. But yea, pretty nifty!